tests/ui/invalid_pyclass_args.stderr

```diff
@@ -1,223 +1,356 @@
    |           ^^^
 
+error: invalid format string: expected `'}'` but string was terminated
+  --> tests/ui/invalid_pyclass_args.rs:98:19
+   |
+98 | #[pyclass(str = "{")]
+   |                  -^ expected `'}'` in format string
+   |                  |
+   |                  because of this opening brace
+   |
+   = note: if you intended to print `{`, you can escape it using `{{`
+
+error: invalid format string: expected `'}'`, found `'$'`
+   --> tests/ui/invalid_pyclass_args.rs:102:19
+    |
+102 | #[pyclass(str = "{$}")]
+    |                  -^ expected `'}'` in format string
+    |                  |
+    |                  because of this opening brace
+    |
+    = note: if you intended to print `{`, you can escape it using `{{`
+
+error: The optional string format shorthand argument to `str` is incompatible with any renaming via `name` or `rename_all`.  You should remove the string format shorthand argument and implement the `Display` trait or implement `__str__` directly.
+   --> tests/ui/invalid_pyclass_args.rs:126:29
+    |
+126 | #[pyclass(name = "aaa", str="unsafe: {unsafe_variable}")]
+    |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: The optional string format shorthand argument to `str` is incompatible with any renaming via `name` or `rename_all`.  You should remove the string format shorthand argument and implement the `Display` trait or implement `__str__` directly.
+   --> tests/ui/invalid_pyclass_args.rs:132:29
+    |
+132 | #[pyclass(name = "aaa", str="unsafe: {unsafe_variable}")]
+    |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: The optional string format shorthand argument to `str` is incompatible with any renaming via `name` or `rename_all`.  You should remove the string format shorthand argument and implement the `Display` trait or implement `__str__` directly.
+   --> tests/ui/invalid_pyclass_args.rs:137:15
+    |
+137 | #[pyclass(str="unsafe: {unsafe_variable}")]
+    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: The optional string format shorthand argument to `str` is incompatible with any renaming via `name` or `rename_all`.  You should remove the string format shorthand argument and implement the `Display` trait or implement `__str__` directly.
+   --> tests/ui/invalid_pyclass_args.rs:143:52
+    |
+143 | #[pyclass(rename_all = "SCREAMING_SNAKE_CASE", str="{a_a}, {b_b}, {c_d_e}")]
+    |                                                    ^^^^^^^^^^^^^^^^^^^^^^^
+
+error: No member found, you must provide a named or positionally specified member.
+   --> tests/ui/invalid_pyclass_args.rs:150:15
+    |
+150 | #[pyclass(str="{:?}")]
+    |               ^^^^^^
+
+error: No member found, you must provide a named or positionally specified member.
+   --> tests/ui/invalid_pyclass_args.rs:157:15
+    |
+157 | #[pyclass(str="{}")]
+    |               ^^^^
+
+error: string formatter shorthand cannot be used with enums, please implement `Display` or `__str__` directly
```

Comment:
`The format string syntax cannot be used with enums.` (Similar to the above?) At least we should be consistent with referring to the format string syntax in all error messages and the docs.